lms/envs/common.py

```diff
@@ -965,7 +965,7 @@
     # .. toggle_implementation: DjangoSetting
     # .. toggle_default: False
     # .. toggle_description: Whether to display honor code agreement for learners before their first grade assignment
-    # (https://github.com/edx/edx-name-affirmation)
+    # (https://github.com/openedx/edx-name-affirmation)
```

Comment:
```suggestion
    # (https://github.com/edx/edx-name-affirmation)
```

Reply:
@sarina you need to take this suggestion.

Reply:
ack, missed that one!